acceptance/openstack/loadbalancer/v2/loadbalancers_test.go

```diff
@@ -100,6 +100,12 @@ func TestLoadbalancersCRUD(t *testing.T) {
 
 	tools.PrintResource(t, newListener)
 
+	// L7 policy
+	policy, err := CreateL7Policy(t, client, listener, lb)
```

Comment:
`s/client/lbClient/`

Also, since `policy` isn't being used elsewhere, Go is going to complain. So you can either change this line to:

```go
_, err := CreateL7Policy(t, client, listener, lb)
```

Or add something like this after the err check:

```go
tools.PrintResource(t, policy)
```